Clean up onsite attendance UI

willgearty opened the pull request:
This cleans up the onsite attendance UI in a number of ways:

1. This moves the "section attendance" button on the landing page to above the dropdown menu. I've also changed the text to make it clear that the two are an "either or" situation (before people were trying to use the dropdown and then immediately click the button).
2. Wrap the tables in collapsible divs. From my experience looking at this page for larger programs, these tables can be obnoxiously large, so now they are hidden by default and can be revealed when necessary.
3. Make the tables sortable, because why not?

Fixes https://github.com/learning-unlimited/ESP-Website/issues/3007 and fixes https://github.com/learning-unlimited/ESP-Website/issues/3008.

kkbrum commented on esp/templates/program/modules/onsiteattendance/attendance.html:
```diff
@@ -5,6 +5,12 @@
+
+{% block xtrajs %}
+    {{ block.super }}
+    <script src="/media/scripts/sorttable.js"></script>
```
maybe add a note to the top of student attendance page after the explanation saying "All tables are sortable by the columns if you click on the headers." since its not super obvious that this is a feature but it is very cool and helpful

willgearty commented:
OK, those two things should be addressed. I've also made the section attendance tables sortable, made the "Back to Attendance Summary" button go to a specific timeslot if one was selected on the section attendance page, removed "Student" from a lot of the column headers when it was implied already, and split out the student names to first and last names.
